tests/integration-tests/framework/fixture_utils.py

```diff
@@ -99,6 +99,7 @@ def release(self):
                 self._save_fixture_data(data)
                 return
 
+        timeout = time.time() + 4*30*30  # 4 hours from now
```

Comment:
Issue: `4*30*30` is 60 minutes. Am I missing something?

Reply:
Sorry wrong cut and paste, fixed!